pylib/mozhginfo/mozhginfo/pushlog_client.py

```diff
@@ -50,17 +52,18 @@ def query_revisions_range(repo_url, from_revision, to_revision, version=2, tipso
     req = requests.get(url)
     pushes = req.json()["pushes"]
     # json-pushes does not include the starting revision
-    revisions.append(from_revision)
+    pushs_list.append(query_push_by_revision(repo_url, from_revision))
+    # revisions.append(from_revision)
```

Comment:
If you mean the *revision.append(from_revision)* comment, My answer is no :)  I will remove it and some more comments need been changed before this PR been merged.
But we need query and append the starting revision push because json-pushes does not include the it.